man/io_uring_enter.2

```diff
@@ -44,6 +44,12 @@ number of events in
 before returning. This flag can be set along with
 .I to_submit
 to both submit and complete events in a single system call.
+If this flag is set the thread issuing the syscall must either be the
```

Comment:
Minor clarification here - it is possible to have another thread create it, if it was created in a disabled state with ```IORING_SETUP_R_DISABLED```. Then it just requires the right thread to enable the ring with ```io_uring_enable_rings()```. So either one of those must be true:

1) Created by the thread that is going to use it, or
2) Created by another thread in a disabled state, and enabled by the thread that is going to use it